remote/README.md

```diff
@@ -0,0 +1,49 @@
+# Remote
+
+At Icalia Labs we are strong believes to have a remote culture for every member at team, and we are always finding ways to improve and promote the culture among everyone regarding their location.
+
+## Table of contents
+
+* [Guidelines](#guidelines)
+* [Tools](#tools)
+
+### Guidelines
+
+Here are a couple of guidelines that every member at Icalia who is remote must follow:
+
+* Say hi to everyone on the mornings, #general channel is a good fit for this.
+* Be online at Slack at all times, or at least when you are available
+* Don't take to long to respond messages, 2 hours may be a good time frame
```

Comment:
what about the following?  
- user my name to let me know you are addressing me.
- mention me (@username) to let me know you are addressing me and you need my attention as soon as possible.

Why? why 2hrs? why not 1hr? half hour? what is the point of this number? what happen if it takes me 2hrs and 2 minutes? etc...

Reply:
By this `user my name to let me know you are addressing me` you mean direct message?

Yeah, I just thought of two hours os maybe 25 minute(a'la pomodoro) just for the people not to expect and throw a lot of messages interrupting and maybe breaking the flow, what is reasonable for you?

Reply:
@kurenn this one may get trickier, what'd happen if we ever got someone that is in a completely different time zone (like 10 hours difference)? , it'll be not cool (AT ALL) to ask him/her to be online at the same time we Mexicans are (or maybe a closer scenario in which one of us wants to travel to a country with a different time zone and work from there)

For that, and also for the reason that @thelastinuit is mentioning, I don't think is necessarily a requirement for us to be always on slack. The whole thing of being remote is to be able to work _asynchronously_, so I'm more inclined to:

- Having a tool in which we're able to communicate with each other (DM's in slack are fine, but without a time frame to respond, also for this purpose, I think the premium version of slack will be mandatory)

Reply:
Agree on that, I think just to be online whenever you are available, in other words if you are online I can ask you things, and expect a response, isn't?

Why do you think premium is mandatory?

Reply:
hmm will avoid DM to avoid fragmented communication (especially for more than two in a team). I made a mistake: "use my name to let me know..." as in

#vamonos: "y'all cejas do you know this shit or that shit?" it means please answer me when you can

#vamonos: "y'all @cejas do you know this shit or that shit?" it means please answer me as soon as possible

That conversation in DM could happen but perhaps someone else may or may not find the value in it. If you are worried about the SPAM, then after that comment, people can thread it and in so keep the channel cleaner.

does this make sense?

Reply:
It makes sense, can you place it on the PR?

Reply:
> Why do you think premium is mandatory?

@kurenn well we'll need a tool for leaving messages right? One that allows leaving a message to someone and we also need to make sure that person _reads_ the message. Github uses github issues for, same goes with Gitlab they use issues to leave a message for the rest of the team. Slack is a tool that feels natural for Icaliers, but there'll be a problem if the messages start to disappear (since the quantity in the free version is restricted to only an amount of them)

Reply:
Agreed with @mayra-cabrera A temporary solution is too pin important msg/posts. I don't know how many of them you can have, though.

Reply:
How long a message is valid to stay on slack?, I mean if you did not read the message from a week it is totally irrelevant at that point isn't?

What type of messages do you have in mind that could fit in that scenario?

Reply:
hmmm true that.

Reply:
> How long a message is valid to stay on slack?, I mean if you did not read the message from a week it is totally irrelevant at that point isn't?

Well not the scenario that I was intended for, maybe I didn't explain myself so here it goes again: We need a tool that allows communication in an asynchronous way (_asynchronous way = Icaliers aren’t always online together at the same time, maybe because they're not in the same time zone... I know this is not a current scenario but bear with me, I swear I have a point_). 

When communicating asynchronously, we need to keep in mind that people need more information than communicating in a synchronous way, because there's a chance that you won't be around to answer important follow-up questions. So, when done asynchronously, it could happen that conversations that would take 5 minutes in real time can take literal days. So what would happen if that information/conversation gets deleted (because Slack is only allowing a certain amount of messages)? Well it would disappear, and If it disappears that means that **it never happened**  

It's the same scenario as commits, issues & pull requests: when we're programming and we found a method or a piece of code that does not make much sense, what we usually do (or maybe just me) is to navigate through the commits log and see why it was added. Also searching  through GitHub pull requests/issues is really useful, maybe there's even a conversation that I could reference and help me understand why it was designed that way. If we didn't have that, well basically we would be on our own 😓 


So long story short, we need a tool that allows us to reference/store messages without deleting them every month or so. 

My two cents 💰

Reply:
Also, i'm not sure but slack removes/hides information from days ago OR quantity(?), if so, what would happen if that particular chat/channel/etc has a lot of content in matter of minutes? it would be lost! IT WILL VIOLATE the most basic law in the universe: conservation of information!!

Reply:
@mayra-cabrera It really makes sense, but I don't know if Slack is then even the right tool, apart from being premium or not, maybe a board messages? could basecamp work?

Reply:
Wohhaaa sorry for the delayed response! Yeah I was thinking of slack because if feels natural to use it, but another tool like basecamp could work! I even got my eye on Github issues 😉

Reply:
The thing with GitHub issues is that we cannot include Mel or Pablo to post messages or persons that are not yet at the company

Reply:
Ok, maybe we can try with basecamp and leave a an issue open to discuss about it?

Reply:
I made some change, you can see those changes at https://github.com/IcaliaLabs/guides/pull/64/commits/b44b7276479a5e30632a0640f2ff0c05778a37af

Reply:
Sure! I like basecamp :)

Reply:
What about pining the messages on slack? for what I can see they are not removed from the log in slack, you can actually check it out with the #general channel at our slack, there is a message from edolopez from Feb, 2016.

Do you think pinned messages could work?

Reply:
I think that would be a good first approach, although I believe things can get complicated when we have like a "gazillion" pinned messages... But I guess it's worth a try 👼

Reply:
My idea is to add or remove the pinned messages as necessary, and i think anyone can unpin, but yeah let's try that